source/fundamentals/crud/write-operations/insert.txt

```diff
@@ -7,6 +7,64 @@ Insert a Document
 Overview
 --------
 
-In this section, we show you how to call the write operations to **insert**
-documents from a collection in your MongoDB database.
+In this guide, you can learn how to insert documents into your MongoDB
+database. To insert a document, use insert operations.
 
+Insert operations consist of the ``insertOne()`` and ``insertMany()``
+methods. For both methods, the driver automatically generates a unique
+``_id`` for documents unless specified. 
```

Comment:
```suggestion
``_id`` for documents unless the document specifies an ``_id``. 
```

In the past, I have been frustrated with documentation that references an action but does not give me a clear description of how to perform that action. When I here, "unless specified," my next thought is "how do I specify that?"